lib/Controllers/create.js

```diff
@@ -17,7 +17,29 @@ Create.prototype.plurality = 'plural';
 Create.prototype.write = function(req, res, context) {
   context.attributes = _.extend(context.attributes, req.body);
 
-  var self = this;
+  var self = this,
```

Comment:
well it seems you can at least cache the foreign keys, and not have to recalculate every time